go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/canonical/jimm/v3
 
-go 1.22.5
+go 1.23
+
+toolchain go1.23.2
```

Comment:
The [`go.mod`](https://github.com/juju/terraform-provider-juju/blob/main/go.mod) file in the Juju Terraform Provider has two `require` clauses (both with direct dependencies), and puts the Juju module (with a comment pointing to the Juju version) in the first one. I think it's a bit nicer to do the same thing in JIMM:

```
module github.com/juju/terraform-provider-juju

go 1.21

require (
	github.com/bflad/tfproviderlint v0.30.0
	github.com/hashicorp/terraform-plugin-docs v0.19.4
	// v3.5.3
	github.com/juju/juju v0.0.0-20240724081236-63d460f9ee6c

)

require (
	github.com/canonical/jimm-go-sdk/v3 v3.0.4
	github.com/dustin/go-humanize v1.0.1
	github.com/hashicorp/terraform-json v0.22.1
    ...
```

Reply:
I'm not understanding why?

Reply:
Just to have Juju near the top of the `go.mod` file, not in the middle the deps list. Anyway, this is just a suggestion, so feel free to ignore.

Reply:
Done it!